tools/cldr-apps/src/main/webapp/DataExport.jsp

```diff
@@ -17,8 +17,12 @@
         response.sendRedirect(request.getContextPath());
         return;
     }
-
-    if (action.equals("list") && UserRegistry.userIsVetter(sess.user)) {
+    if (action.equals("flagged") && UserRegistry.userIsTC(sess.user)) {
```

Comment:
Given the need to migrate away from JSP, it would be more ideal for new code like this to go into a Java file

Reply:
@btangmu ideally, but it's parallel to existing code

Reply:
@btangmu also, this is just a first change, @macchiati to identify other places where we should do this same download.

Reply:
OK, maybe while this code is fresh in your mind and you're still working on this ticket would be a good time to move all of DataExport.jsp into a .java file :-)